include/swift/Basic/BasicBridging.h

```diff
@@ -214,7 +214,8 @@ void BridgedData_free(BridgedData data);
 //===----------------------------------------------------------------------===//
 
 enum ENUM_EXTENSIBILITY_ATTR(open) BridgedFeature {
-#define LANGUAGE_FEATURE(FeatureName, SENumber, Description) FeatureName,
+#define LANGUAGE_FEATURE(FeatureName, IsAdoptable, SENumber, Description)      \
```

Comment:
Could we add some documentation how to determine if my feature should be `IsAdoptable`? Sometimes I get to these defines like "once a year" and have to search around how to even determine what the value here should be driven by, even some minor docs would help 🙏

Reply:
Good idea, will add a more detailed parameter description in the def file.